assets/test/socket_test.js

```diff
@@ -391,6 +391,19 @@ describe("channel", () => {
     const [foundChannel] = socket.channels
     assert.deepStrictEqual(foundChannel, channel)
   })
+
+  it("doesn't add duplicated channel if already exists", () => {
+    assert.equal(socket.channels.length, 0)
+
+    channel = socket.channel("topic", {one: "two"})
+    assert.equal(socket.channels.length, 1)
+
+    channel = socket.channel("topic", {two: "three"})
+    assert.equal(socket.channels.length, 1)
+
+     const [foundChannel] = socket.channels
```

Comment:
looks like a redundant space

Reply:
Fixed @achempion :-)
Thanks for the review!